server/src/teams/teamController.js

```diff
@@ -18,33 +14,87 @@ const addTeam = (req, res) => {
     }
   }
 
-  request(options, (err, response, body) => {
-    body = JSON.parse(body);
-    let slackTeamData = {
-      slackTeamToken: body.access_token,
-      slackTeamName: body.team_name,
-      slackTeamId: body.team_id,
-      slackBotId: body.bot.bot_user_id,
-      slackBotToken: body.bot.bot_access_token
-    }
+  rp(options)
+    .then((body) => {
+      body = JSON.parse(body);
+      let slackTeamData = {
+        slackTeamToken: body.access_token,
+        slackTeamName: body.team_name,
+        slackTeamId: body.team_id,
+        slackBotId: body.bot.bot_user_id,
+        slackBotToken: body.bot.bot_access_token
+      }
 
-    if (body.ok) {
-      Team.findOrCreate({ where : slackTeamData })
-        .spread( (team, created) => {
-          res.redirect('/');
-        })
-        .catch(err => {
-          //TODO: redirect to error page
-          res.redirect('/');
-        })
-
-    } else {
-      //TODO: redirect to error page
-      res.redirect('/');
-    }
+      if (body.ok) {
+        Team.findOrCreate({ where : slackTeamData })
+          .spread( (team, created) => {
+            findTeamUsers(team);
+            res.redirect('/');
+          })
+          .catch(err => {
+            //TODO: redirect to error page due to bad data base request
+            res.redirect('/');
+          })
+      } else {
+        //TODO: redirect to error page due to failed respone request to slack
+        res.redirect('/');
+      }
 
-  });
+    })  
+    .catch(err => res.redirect('/') /* due to failed request to slack */ );
 
 } 
 
+
+const findTeamUsers = (team) => {
+  let teamId = team.dataValues.id;
+  let token = team.dataValues.slackBotToken;
+
+  let teamUsersData = {
+    uri: 'https://slack.com/api/users.list',
+    method: 'GET',
+    qs: { token }
+  }
+
+  rp(teamUsersData)
+    .then(body => {
+      body = JSON.parse(body);
+
+      if (body.ok) {
+
+        let users = parseUsersInfo(body.members);
+
+        let usersData = { 
+          url: 'http://localhost:8080/slack/users',
+          method: 'POST',
+          json: { users, teamId } 
+        }
+
+        rp(usersData)
+        .then(body => console.log(body))
```

Comment:
send something back from the /slack/users request so we can confirm that the data was posted properly to the users endpoint